modules/vaos/config/routes.rb

```diff
@@ -2,7 +2,9 @@
 
 VAOS::Engine.routes.draw do
   defaults format: :json do
-    resources :appointments, only: :index
+    resources :appointments, only: :index do
+      put 'cancel', on: :collection
```

Comment:
🤔  I'm back and forth on a separate cancel as opposed to the more semantic destory/DELETE. 

Maybe a refactor towards this being an `AppointmentCancellation` controller if its substantially different from an appointment (which based on the index action it is pretty different)

Reply:
I agree, but those actions typically take an ID associated with the object being deleted. They are member actions. This is a collection action, and there is no actual ID. A synthetic id might be some combination of appointmentTime and clinicID, but the underlying API doesn't treat it at that way, they expect a full JSON payload to be processed.